src/NKSolver/blockette.F90

```diff
@@ -189,9 +189,7 @@ subroutine blocketteRes(useDissApprox, useViscApprox, useUpdateIntermed, useFlow
 
        do sps=1, nTimeIntervalsSpectral
           ! Update overset connectivity if necessary
-          if (oversetPresent .and. &
-               (oversetUpdateMode == updateFast .or. &
-               oversetUpdateMode == updateFull)) then
+          if (oversetPresent .and. oversetUpdateMode == updateFast) then
```

Comment:
This was a bug. `updateOversetConnectivity` is the fast update routine. The docstring makes this clear:
https://github.com/mdolab/adflow/blob/2a7d031e1c56c518ec977bd7d13142c25ac429d6/src/overset/oversetCommUtilites.F90#L1739-L1746
This does not apply to the full update because in that case the entire hole cutting procedure is run again.